fix/point-simulator-test (#2321)
mercadopago/devsite-docs: guides/mp-point/integration-api/point-simulator.en.md

The Point Simulator is a tool that allows you to interact with various endpoints of our API to integrate Point of Sale devices with our system. Its main objective is to help you understand the flow of creating and canceling a payment attempt.

> For this reason, the Point Simulator is a tool that will also allow you to test that your Point integration and payments processing is working correctly. Although it’s not an exact replica of a device, it has educational purposes, to help with familiarization with our API.

Reviewer: ```suggestion
For this reason, the Point Simulator will also allow you to test that your Point integration and payments processing are working correctly. Although it’s not an exact replica of a device, it has educational purposes, to help with familiarization with our API.
```

The simulator has two modes of use:

* **PDV mode**: simulates the integration of a complete system (device and Point of Sale) with our API.

* **Device mode**: simulates a virtual Point of Sale device so you can test your integration using HTTP requests. 

To learn how to use the simulator, follow the steps below.

## 1. Generate the test environment

> To start testing integrations and payment flows with the Point Simulator, you will need to generate a [seller test users](/developers/en/docs/mp-point/additional-content/your-integrations/test/accounts) and access their **production credentials**. The Simulator will ask you for the seller test user test credentials, but you will need the production ones to access their account and settings.

Reviewer: Observación: La parte que dice "The Simulator will ask you for the seller test user test credentials, but you will need the production ones to access their account and settings." podría estar destacada con un mensaje de Important, ya que es una contradicción entre lo que va a decir el Sim y lo que estamos pidiéndole que haga. Creo que valdría la pena destacarlo apenas más.

```
------------

> Please note that when creating a test user for an application already integrated with Point, **a store and a point of sale will be automatically generated and associated with that test seller account**. This means that you can carry out your tests on payment processing without the need for additional configurations. 

Author: En general, cuando es con mayus es porque es el modo PDV, y cuando es minúscula es el equivalente a "caja". Voy a revisar igualmente por si se me pasó alguno o si puedo dejarlo más claro!

> To query a payment made by the simulator through the [Payments API](/developers/en/reference/payments/_payments_id/get), you must use the same test `access-token` that you use in the Simulator.

## 7. Receive the notification

When payment intents are processed, an alert is sent to the URL configured as the notification channel by the integrator. In the case of test payment notifications made through the Simulator, they will be sent to the channel you defined as the notification pathway when configuring your test environment with your seller user. 

> To validate their reception and proper functioning, go to your notification log. There, you will find the sent notification with the transaction status, and you can verify its details.

Reviewer: ```suggestion
To validate their reception and proper functioning, go to your notification log. There, you will find the sent notification with the transaction status, so you can verify its details.
```
